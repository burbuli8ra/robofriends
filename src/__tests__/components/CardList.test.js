import React from 'react';
import { shallow } from 'enzyme';
import CardList from '../../components/CardList';

describe('CardList Component', () => {
  it('should render CardList component', () => {
    const mockRobots = [{
      id: 1,
      name: 'John Snow',
      username: 'John Snow',
      email: 'johnsnow@gmail.com'
    }];

    const wrapper = shallow(<CardList robots={ mockRobots } />);

    expect(wrapper).toMatchSnapshot();
  });
});
