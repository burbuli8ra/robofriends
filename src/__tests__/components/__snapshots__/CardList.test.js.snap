// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList Component should render CardList component 1`] = `ShallowWrapper {}`;
